mkat_fpga_tests/test_cbf.py

```diff
@@ -493,10 +496,10 @@ def test_back2back_consistency(self):
                 chan_responses.append(this_freq_response)
 
             diff_dumps = []
-            for comparison in range(1, len(dumps_data)):
+            for comparison in range(len(dumps_data)):
```

Comment:
No need to compare dump 0 to itself? That is why the range(1, ...) was in there.